resources/vue/src/composables/useGetTrips.js

```diff
@@ -0,0 +1,10 @@
+export default function useGetTrips () {
+    const useGetTrips = async (url, variable) => { //TODO: вынести в тулы
+        let response = await fetch(url),
+            json = await response.json();
```

Comment:
These should be `const` and I also, in general, dislike creating variables with a comma, but it's just my preference, you can safely ignore it :P
```suggestion
        const response = await fetch(url),
            json = await response.json();
```